mklive.sh.in

```diff
@@ -337,6 +355,22 @@ if [ "$(id -u)" -ne 0 ]; then
     die "Must be run as root, exiting..."
 fi
 
+#The -d and -t options are complementary. If one exists, the other must also exist. 
+#If these options are set, I also check sbsign command.
+if  ([ -z $DBKEY ] &&  [ ! -z $DBCRT ]) || ([ ! -z $DBKEY ] &&  [ -z $DBCRT ]); then
+    die "Must be set a key and certificate via -d and -t option, exiting..."
+elif ([ $DBKEY ] && [ $DBCRT ]); then
+    if [ $DBKEY ] && [ ! -f $DBKEY ]; then
+        die "$DBKEY does not exist, exiting..."
+    elif [ $DBCRT ] && [ ! -f $DBCRT ]; then
+        die "$DBCRT does not exist, exiting..."
+    elif ! [ -x "$(command -v sbsign)" ]; then
```

Comment:
What wrong with simple:
```sh
elif command -v sbsign; then
```

Reply:
command -v sbsign >/dev/null
or
hash sbsign 2>/dev/null
or
type sbsign >/dev/null 2>&1

and if im right, then hash is the fastest, but i think ive only ever seen command -v around void

Reply:
I don't care about which one is the fastest. I was talking about the uselessness of `[ -x ]`

Reply:
the point was the redirection against littering the output, while u r right, just these are again two different things, i hope u didnt get me wrong or got offended or whatever! :)

Reply:
Ah, no, I'm not offended. However, driving out of track is not productive.
You suggestion about appending `>/dev/null` is good, that's is nice.

Reply:
ahh, fine, and thx for the info! :) 
however, just for the next time, what do u mean by driving out of track? it was about the same line, is there a way to isolate my stuff better that im not aware of, or i just need a better wording? im not a git/github pro actually :D (also, i often go off topic, but i dont think this was an example of that :D )

Reply:
Actually, this one is not. The previous one is :-p